widgets/WalkmeshManager.cpp

```diff
@@ -437,7 +436,7 @@ QWidget *WalkmeshManager::buildCameraRangePage()
 	layout->addWidget(group1);
 	layout->addWidget(group2);
 	layout->addWidget(group3);
-	layout->addStretch();
+	//layout->addStretch();
```

Comment:
it seams to do nothing. i have left it commented so others can try with / with out it to see a differance.